src/components/Topbar/index.tsx

```diff
@@ -2,11 +2,19 @@ import styled from "styled-components";
 import Link from "../Link";
 import ROUTES from "@constants/routes";
 import APP from "@constants/app";
+import Image from "next/image";
+import Sitelogo from "../../../public/static/images/logo/warslive.png";
 
 const Navbar: React.FC = () => {
   return (
     <Container>
       <Section>
+        <Image
+          src={Sitelogo}
```

Comment:
good practice but just use ``"/static/images/logo/warslive.png"`` instead
and the logo must be in ``svg``